Yoast/yoast-acf-analysis
# Update Yarn.lock file

Remove package-lock.json
We are using the Yarn lock file in Travis configuration, having two of them is unwanted.

## Summary

This PR can be summarized in the following changelog entry:

* _N/A_

## Relevant technical choices:

* Removed the package-lock.json
* Updated the yarn.lock file

## Test instructions

This PR can be tested by following these steps:

* Build the JS files
* Run the tests
* Watch for changes

All above actions should just work

Fixes #
